As a user, I don't expect an evaluation conducted for MARFC comparing RFC forecasts to USGS DV and including the correct time zone shift to fail.

The declaration with WRDS URLs omitted is below. James mentioned it may be a limit to the amount of metadata than can be requested.

In three MARFC evaluations, I've received a different failing location each time. An example exception is noted in https://github.com/NOAA-OWP/wres/issues/258#issuecomment-4024095009. Here is the "Caused by" associated with the third run:

```
Caused by: wres.reading.ReadException: Could not convert a local datetime to UTC for a time-series associated with location, USGS-01467005, because the location metadata for USGS-01467005 was not supplied to the reader. Metadata was available for the following locations: [USGS-01397420, USGS-01410600, USGS-01458500, USGS-01410560, USGS-01413038, USGS-01395000, USGS-01413398, USGS-01454700, USGS-01396060, USGS-01446995, USGS-01407600, USGS-01400010, USGS-01403900, USGS-01407705, USGS-01398102, USGS-01398500, USGS-01399830, USGS-01399670, USGS-01411000, USGS-01455500, USGS-01414000, USGS-01408120, USGS-01467048, USGS-01408000, USGS-01461500, USGS-01400000, USGS-01402540, USGS-01409335, USGS-01412800, USGS-01410500, USGS-01411435, USGS-01399780, USGS-01396790, USGS-01397000, USGS-01457000, USGS-01409125, USGS-01409400, USGS-01407500, USGS-01403540, USGS-01401000, USGS-01403400, USGS-01396660, USGS-01457500, USGS-01411300, USGS-01407290, USGS-01411390, USGS-01405030, USGS-01408029, USGS-01401750, USGS-01402500, USGS-01410000, USGS-01411330, USGS-01407081, USGS-01413088, USGS-01412150, USGS-01407760, USGS-01409146, USGS-01403200, USGS-01396800, USGS-01394500, USGS-01394620, USGS-01410510, USGS-01459500, USGS-01393890, USGS-01411320, USGS-01413500, USGS-01396000, USGS-01403150, USGS-01408168, USGS-01408205, USGS-01409810, USGS-01408048, USGS-01401650, USGS-01408900, USGS-01410784, USGS-01410820, USGS-01399500, USGS-01411355, USGS-01399100, USGS-01410150, USGS-01411360, USGS-01409280, USGS-01406050, USGS-01403060, USGS-01405400, USGS-01402630, USGS-01403600, USGS-01413408, USGS-01396500, USGS-01411500, USGS-01455400, USGS-01398000, USGS-01396582, USGS-01440200, USGS-01411350, USGS-01400500, USGS-01403570, USGS-01407770, USGS-01402000, USGS-01408500]
```

Two earlier runs are noted in #258.

If you need the `predicted` data for the declaration below, let me know. It won't be easy. Thanks,

Hank

=====================

```yaml
label: Example Meal for 6-hour AHPS vs USGS
observed:
  sources:
    - uri: https://api.waterdata.usgs.gov/ogcapi/v0/collections/daily/
      parameters:
        statistic_id: '00001'
      interface: usgs nwis
      daylight_savings: true
  variable: '00065'
  time_shift:
      period: 1
      unit: hours
predicted:
  label: WRDS AHPS Streamflow Forecasts
  sources:
  - interface: wrds ahps
    uri: https://[omitted]/api/rfc_forecast/v2.0/forecast/stage   # URI was corrected to stage
  variable:
    name: HG
unit: ft
rescale_lenience: all
lead_time_pools:
  period: 24
  unit: hours
lead_times:
  minimum: 0
  maximum: 120
  unit: hours
reference_dates:
  minimum: '2025-12-01T00:00:00Z'
  maximum: '2025-12-25T00:00:00Z'
valid_dates:
  minimum: '2025-12-01T00:00:00Z'
  maximum: '2025-12-25T00:00:00Z'

feature_service:
  uri: https://[omitted]/api/location/v3.0/metadata
  groups:
  - group: rfc
    value: marfc
    pool: false

metrics:
- name: root mean square error
- name: pearson correlation coefficient
- name: sample size
- name: volumetric efficiency
- name: quantile quantile diagram
- name: scatter plot
- name: time series plot
duration_format: hours
decimal_format: '0.0000'
output_formats:
- format: csv2
- format: png
- format: pairs
- format: protobuf
```

--- a/wres-reading/src/wres/reading/nwis/ogc/NwisReader.java
+++ b/wres-reading/src/wres/reading/nwis/ogc/NwisReader.java
@@ -126,7 +126,6 @@ public class NwisReader implements TimeSeriesReader
     /** Cache of location metadata for re-use. */
     private final Cache<@NonNull String, LocationMetadata> locationCache =
             Caffeine.newBuilder()
-                    .maximumSize( 100 ) // 100, arbitrarily
                     .build();
 
     /** For reading feature metadata. */

